include/swift/Driver/Job.h

```diff
@@ -230,9 +230,20 @@ class CommandOutput {
 class Job {
 public:
   enum class Condition {
+    // There was no information about the previous build (i.e., an input map),
+    // or the map marked this Job as dirty
+    // but the input didn't change.
```

Comment:
This isn't correct; the job being marked as "dirty + cascading" takes priority over whether or not the input changed. (At least, I hope it does.)

Reply:
Thank you. Fixed!